turbo.json

```diff
@@ -54,6 +55,12 @@
         ".output/**"
       ]
     },
+    "generate:component-statuses": {
+      "cache": false,
```

Comment:
Ideally we should be able to cache this step. Am I right in thinking the only file inputs are the `package.json` files? Could we add those as `inputs` here and then enable caching?

I won't block this PR over it but it might be a nice improvement to make.

Reply:
Lets catch up on this and perhaps push as a future change